fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -693,6 +693,12 @@ public class Config extends ConfigBase {
     @ConfField
     public static int http_max_chunk_size = 8192;
 
+    // Because the new version of netty has a stricter headers validation,
+    // so the validation is turned off here to be compatible with old users
+    // https://github.com/netty/netty/pull/12760
+    @ConfField
+    public static boolean http_validate_headers = false;
```

Comment:
Because this is a boolean configuration it should be start with enable_

Reply:
done